### PR TITLE
fix(model): reverser animation

### LINKED ISSUE #1874: [BUG] Engine Reverser
**Mod Version**

**Describe the bug**

Why is the cowl opening degree synchronized to the level of reverse thrust? it should be fully opened despite its thrust level. hope this will be changed.

**To Reproduce**

1.(set some axis for reverser thrust via FSUIPC)
2.see it from outside view
3.

**Expected behavior**

it should be fully opened despite its thrust level. 

**Actual behavior**

cowl opening degree synchronized to the level of reverse thrust

**References**

https://www.youtube.com/watch?v=uUw_HLNu2ZA

**Additional context**

Was this working before/when did the issue start occurring?
no
Is this a problem in the vanilla unmodded game?
yes

Discord username (if different from GitHub):

### LINKED ISSUE #5050: [POWERPLANT] REV LOGIC
### Aircraft Version

Development

### Build info

```json
{
  "built": "2021-06-07T23:11:33+00:00",
  "ref": "merge",
  "sha": "1d041a6aec11684fe0402a47c698ad303d3d782c",
  "actor": "aguther",
  "event_name": "manual"
}
```

### Describe the bug

Issue 1

Revs able to be deployed in-flt.

Issue 2 

Rev shells opening according to thrust lever position (Up to 20%)

### Expected behavior

Issue 1

Not supposed to happen

Issue 2

Once REV GREEN is displayed on the N1 Dial, the reverse shell is already fully deployed, that is, for Idle reverse, a fully opened shell is expected.

### Steps to reproduce

Issue 1

Deploy rev in-flt

Issue 2

1. Set thrust levers between 0% REV and 20% REV (Thus beyond Idle REV)
2. Rev shell will be gradually opening (not supposed to happen)

### References (optional)

https://www.youtube.com/watch?v=WA5UGY4uPO8

### Additional info (optional)

rev

### Discord Username (optional)

Illuminati

### PR DESCRIPTION
**Pull request by tracernz**
Fixes #1874
Fixes #5050

## Summary of Changes

Adjust reverser animation so it's no longer tied to reverse lever percent. It now opens in 2 seconds whenever reverse thrust is engaged, and closes in 2 seconds whenever reverse thrust is disengaged.

## Screenshots (if necessary)

https://youtu.be/drINS8HAZ-E

## References

https://youtu.be/WdMRFM5tOdE?t=339
I am judging that reversers take approx 2 seconds to open/close from this video.

## Additional context

Discord username (if different from GitHub):

## Testing instructions

Apply reverse idle, check that reverser fully opens with smooth animation, apply idle (forward) and check that reverser smoothly closes.
Apply full reverse, check that reverser fully opens with smooth animation, apply idle (forward) and check that reverser smoothly closes.

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

File: flybywire-aircraft-a320-neo/ModelBehaviorDefs/A32NX/A32NX_Exterior.xml [hunk 1]
```diff
@@ -94,6 +94,38 @@
             </True>
         </Condition>
     </Template>
+    <!-- Animate a turbine thrust reverser (ID is used to identify which engine to look for) -->
+    <Template Name="A32NX_ENGINE_Turbine_ReverseThrust_Template">
```
Comment (Benjozork) [inline flybywire-aircraft-a320-neo/ModelBehaviorDefs/A32NX/A32NX_Exterior.xml:98]: Templates should start with `FBW_`

Comment (tracernz) [reply flybywire-aircraft-a320-neo/ModelBehaviorDefs/A32NX/A32NX_Exterior.xml]: Fixed. 👍

**Comment by tracernz** [comment]
RiiR (rewriten in RNP)

**Review by Benjozork — APPROVED** [review]

**Comment by Cryptic56** [comment]
Quality Assurance Tester/Trainee Report

Discord: Cryptic#4731
Object of testing: #4381 
Tier of Testing: 1
Date: 03/06/2021

Testing Process:

 Loaded directly onto the runway with aircraft started
 Applied minimum/idle reverse thrust for the animation to activate
reverted to idle thrust.
Applied maximum reverse thrust waited for the animation to activate and then reverted to idle.

Negatives: N/A
Testing Results: Passed

Conclusions: Worked flawlessly, from what I saw it worked exactly as intended and had no hiccups. Great work!
Media: 
Microsoft Flight Simulator Screenshot 2021 06 03 - 18 05 11 17
Microsoft Flight Simulator Screenshot 2021 06 03 - 18 05 23 30
Microsoft Flight Simulator Screenshot 2021 06 03 - 18 05 49 67

**Comment by AviationSpotter** [comment]
Non QA Associated Testing Report

Discord: Aviation Spotter#9102
Object of testing: #4381
Tier of Testing: 1
Date: 3/6/2021

Testing Process: 

1. Tested with engines on
2. Tested with engines off
3. Tested with engines on and during a landing

Negatives: 

• No reverse yellow to indicate reverse doors in transition (for another PR as for this PR is just for the animation)
• During idle reverse thrust, reverse green is given although the reverse doors have not been opened (see in media)

Testing Results: Passed
Conclusions: As far as issues, the issues are unrelated to the animation and more related to the upper ECAM display. 
Media: 

https://user-images.githubusercontent.com/72807945/120703183-faf3b400-c482-11eb-8801-7f8ac19c213a.mp4

**Comment by marcman86** [comment]
Quality Assurance Tester Report

Discord:marcman86#4907
Object of testing: #4381
Tier of Testing: 1
Date: 04/06/2021

Testing Process: 
- Spawned on runway at EDDP
- Reverse idle
- Idle
- Reverse full
- Idle

Negatives: -
Testing Results: Passed
Conclusions: Smooth animation, about 2s. Works.
Media: -
Hope this helps :-)

**Comment by Mack674** [comment]
Quality Assurance Tester Report

Discord: Mack674#2274
Object of testing: PR #4381
Tier of Testing: 2
Date: 05.06.2021

Testing Process: 
- Loaded C&D at gate
- Started up
- Checked animations
- Taxied to the runway
- Checked reversers with rejected takeoff

Negatives: N/A
Testing Results: Passed 

Conclusions: Very good work. Animations works very smooth and activated when it should.

**Comment by DavidEGTE** [comment]
Quality Assurance Trainee Report

Discord: David#2177
Object of testing: #4381 
Tier of Testing: 1
Date: 08/06/2021

Testing Process: 
C&D @ LIRN
SOP until the line
Taxied to the runway
Checked reversers with rejected take-off
Checked animations

Negatives: N/A
Testing Results: Passed

Conclusions: LGTM

**Pull request merged**